contracts/core/Registry.sol

```diff
@@ -23,30 +22,42 @@ contract Registry is AccessControl {
     struct IdentityDetails {
         string name;
         Metadata metadata;
-        address attestationAddr;
+        address anchor;
```

Comment:
I do prefer `attestationAddress` cause it's more informative and goes without idea of have EAS as first class citizens

Reply:
How about `attestor`?

Reply:
Hmm this wouldn't be the attestor, this is the subject being attested :P

Reply:
Ah good point

Reply:
I think sticking to anchor works for now. It does feel vague but better than the other options